packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/ProvenanceQueryParameters.java

```diff
@@ -0,0 +1,23 @@
+package software.uncharted.terarium.hmiserver.models.dataservice;
+
+import lombok.Data;
+import lombok.experimental.Accessors;
+
+import javax.json.bind.annotation.JsonbProperty;
+import java.io.Serializable;
+
+@Data
+@Accessors(chain = true)
+public class ProvenanceQueryParameters implements Serializable {
+
+	@JsonbProperty("root_id")
```

Comment:
In its current form, these annotations aren't doing anything as the java objects are also named with underscores.  These objects should be renamed to camel case (for example`rootId`).

If you want to access these properties by camel case in the front end instead of the underscored version, you would do something like...

```
private Number rootId;
.
.
.
@JsonbProperty(root_id)
public void setRootId(Number rootId){
     this.rootId = rootId;
}
```

Reply:
Good point as I noticed some other places in the code such as xdd_uri was not missed and still uses the underscore. I will fix both. Thanks